rsyslog/Sanity/gnutls-openssl/main.fmf

```diff
@@ -0,0 +1,49 @@
+summary: a sanity test for TLS communication
+contact: Dalibor Pospíšil <dapospis@redhat.com>
+test: ./runtest.sh
+require+:
+- library(distribution/dpcommon)
+- library(ControlFlow/ConditionalPhases)
+recommend:
+- rsyslog
+- gnutls
+- gnutls-utils
+- rsyslog-gnutls
```

Comment:
Add rsyslog-openssl package also

Reply:
sure I just did not know what the name will be